Makefile

```diff
@@ -279,7 +283,7 @@ test: run-tests
 run-tests:
 ## run-tests: Run the unit tests
 	$(eval TMP := $(shell mktemp -d $${TMPDIR:-/tmp}/jj-XXX))
-	$(eval TEST_PACKAGES := $(shell go list $(PROJECT)/... | grep -v $(PROJECT)$$ | grep -v $(PROJECT)/vendor/ | grep -v $(PROJECT)/acceptancetests/ | grep -v $(PROJECT)/generate/ | grep -v mocks))
+	$(eval TEST_PACKAGES := $(shell go list $(PROJECT)/... | sort | ([ -f $(TEST_PACKAGE_LIST) ] && comm -12 $(TEST_PACKAGE_LIST) - || cat) | grep -v $(PROJECT)$$ | grep -v $(PROJECT)/vendor/ | grep -v $(PROJECT)/acceptancetests/ | grep -v $(PROJECT)/generate/ | grep -v mocks))
```

Comment:
I think you can combine all the greps into a list with `[]` might read a bit better then?

Reply:
I tried this and it was more messy but less verbose, so sticking with the current one.